compiler/parser/chapel.ypp

```diff
@@ -595,6 +598,30 @@ opt_prototype:
 | TPROTOTYPE { $$ = true;  }
 ;
 
+/*include_access_control:
+  TINCLUDE { $$ = false; @$.comment = context->latestComment; context->latestComment = NULL; } // public
+| TINCLUDE TPUBLIC { $$ = false; @$.comment = context->latestComment; context->latestComment = NULL; } // also public
+| TINCLUDE TPRIVATE { $$ = true; @$.comment = context->latestComment; context->latestComment = NULL; } // private
+;*/
```

Comment:
Should this be removed as part of clean-up?  Or if it has value, add a comment indicating what that is?

Reply:
Yeah I'll remove that